contributors/contributors/community.md

```diff
@@ -121,21 +125,21 @@ Community Coordinator: TBD
 
 #### [Knowledge Base Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
 
-Knowledge Base Contributors share feedback, questions and answers on Mattermost through forums for [trouble shooting](https://www.mattermost.org/troubleshoot/), [feature proposals](https://www.mattermost.org/feature-ideas/), Github issues and the [Mattermost community server](https://community.mattermost.com/).
+Knowledge Base Contributors share feedback, questions, and answers on Mattermost through forums for [trouble shooting](https://www.mattermost.org/troubleshoot/), [feature proposals](https://www.mattermost.org/feature-ideas/), GitHub issues, and the [Mattermost community server](https://community.mattermost.com/).
 
 Community Coordinator: Amy Blais
 
 #### [Deployment Solution Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
 
-Deployment Solutions Contributors create packaging and automation for making it easier to deploy Mattermost, using tools such as Docker, Kubernetes and other leading virtualization and orchestration solutions. See [Deployment Solutions Programs](https://docs.mattermost.com/guides/orchestration.html) for guidance on how to build to common Mattermost standards.
+Deployment Solutions Contributors create packaging and automation for making it easier to deploy Mattermost, using tools such as Docker, Kubernetes, and other leading virtualization and orchestration solutions. See [Deployment Solutions Programs](https://docs.mattermost.com/guides/orchestration.html) for guidance on how to build to common Mattermost standards.
 
-Community Coordinator: Amy Blais
+Community Coordinator: Jason Blais and Amy Blais
 
 #### [UX Design Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
 
 UX Design Contributors influence the design of the Mattermost user experience through discussion in the `ux-design` channel on the Mattermost continous integration server at [https://community.mattermost.com/core/channels/ux-design](https://community.mattermost.com/core/channels/ux-design) and through contributions to the [Mattermost User Experience Guidelines](https://docs.mattermost.com/developer/fx-guidelines.html) on [GitHub](https://github.com/mattermost/docs/blob/master/source/developer/fx-guidelines.rst).
```

Comment:
@andrewbrown00, would you be open to reviewing this section and confirming that the links are correct?